modules/common/package.json

```diff
@@ -34,6 +35,7 @@
   "devDependencies": {
     "@bugsnag/js": "7.16.1",
     "@erento/eslint-plugin-erento-rules": "1.0.0",
+    "@nestjs/axios": "1.0.0",
     "@nestjs/common": "8.3.0",
     "@nestjs/core": "8.3.0",
```

Comment:
Aren't those dependencies, runtime dependencies? (Also bugsnag)...

Reply:
for this service it is required as a minimal version... but host applications can use higher version defined in peer deps